README.md

```diff
@@ -110,6 +109,31 @@ Currently, the available columns are:
 | finalize promoted (mb) | The size of finalizable objects that were discovered to be dead and so promoted during this GC, in MB.                                                                           | `TraceGC.HeapStats.FinalizationPromotedSize / 1000000.0`                                  |
 | pinned objects         | Number of pinned objects observed in this GC.                                                                                                                                    | `TraceGC.HeapStats.PinnedObjectCount`                                                     |
 
+## Call Stacks
+
+In an effort to aid with perf diagnosis, call stacks are currently available for induced GCs and large object allocations on Windows, only. Enabling call stacks requires the process to run with administrator privileges.
+
```

Comment:
I would probably also add that there might be a significant overhead when you are asking for stacks, just so that users are not surprised.